lua/lualine/components/filename.lua

```diff
@@ -24,6 +28,9 @@ end
 
 FileName.update_status = function(self)
   local data = vim.fn.expand('%:p')
+  local winW = vim.fn.winwidth(0)
+  local estimatedSpaceAvailable = winW - 40
```

Comment:
We use snake case in the repo . So please follow the style.

Also shouldn't it be just 40 ? doesn't winW - 40 mean it'll leave 40 chars for all the other component and itself shorten when it can't prov9de that ?

Reply:
I thought this was the window width, so the estimated space would be dynamically 40 less than the width of the window

Reply:
yes it's window width . What I was saying was previously it would've shorted when the filename > 40 in all windows sizes . Now it's when filename > 80 for winW 120 filename > 20 for winW 60 . It's not nesserily a bad thing . I was just asking  if you were trying to replicate previous behavior :)